source/grid/grid_out.cc

```diff
@@ -3442,27 +3480,41 @@ namespace internal
                     {
                       for (unsigned int l=0; l<GeometryInfo<dim>::lines_per_face; ++l)
                         {
+                          point_cache.clear();
```

Comment:
This likely also applies above: if you clear the cache here, then you may as well declare the variable here. I think that might make the code easier to read, given that it's then clear that the cache only applies to one line.

I know that you did this for performance reasons, but I'm not terribly concerned about performance in this function. Are you?

Reply:
The function is dominated by IO time; I moved the vector into this scope.